code/src/components/GameBoard.js

```diff
@@ -0,0 +1,178 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { nextStage } from 'reducers/labyrinth';
+import styled from 'styled-components/macro';
+import TypeWriterEffect from 'react-typewriter-effect';
+import sleepingPotato from 'assets/sleeping-potato.gif';
+import { Container } from './styling/GlobalWrappers';
+
+const GameBoard = () => {
+  const dispatch = useDispatch();
+  const { description, actions, coordinates } = useSelector((store) => store.labyrinth.stage);
+  const userName = useSelector((store) => store.labyrinth.username);
+
+  const onDirectionButtonClick = (type, direction) => {
+    dispatch(nextStage(type, direction));
+  };
+
+  const onRestartButtonClick = () => {
+    return window.location.reload();
+  };
+
+  const winner = () => {
+    return (
+      <>
+        <WinnerWrapper>
+          <div>
+            <h2>YOU WIN!</h2>
+            <img src={sleepingPotato} alt="You win" />
+            <p>
+              Good job <span>{userName.toUpperCase()}</span>, you managed to get Potato back home
+              safely.
+            </p>
+          </div>
+        </WinnerWrapper>
+        <RestartBtn type="button" onClick={() => onRestartButtonClick()}>
+          RESTART
+        </RestartBtn>
+      </>
+    );
+  };
+  return (
+    <>
+      <Top hide={coordinates === '1,3'}>
+        <ButtonAndCompass>
+          <RestartBtn type="button" onClick={() => onRestartButtonClick()}>
+            RESTART
+          </RestartBtn>
+        </ButtonAndCompass>
+        <p>Player: {userName}</p>
+      </Top>
+      <Container>
+        <Description>
+          <TypeWriterEffect
+            startDelay={100}
+            cursorColor="#00cf00"
+            text={description}
+            typeSpeed={70}
+            hideCursorAfterText />
+        </Description>
+        <MazeImgContainer hide={coordinates === '1,3'}>
+          <ImgNavigation>
+            {actions
+              && actions.map((item) => {
+                return (
+                  <ButtonWrapper key={item.direction}>
+                    <button
+                      type="button"
+                      onClick={() => onDirectionButtonClick(item.type, item.direction)}>
+                      GO {item.direction.toUpperCase()}
+                    </button>
+                  </ButtonWrapper>
+                );
+              })}
+          </ImgNavigation>
+        </MazeImgContainer>
+        <ActionsWrapper>
+          {actions
+            && actions.map((item) => {
+              return (
+                <div key={item.description}>
+                  <Pdirection>WHAT DO YOU SEE TO THE {item.direction.toUpperCase()}:</Pdirection>
+                  <p>{item.description}</p>
+                </div>
+              );
+            })}
+        </ActionsWrapper>
+        {coordinates === '1,3' && winner()}
```

Comment:
We noticed that you made "winner" a function and it is called here. It is so amazing how we all can make the same result show using different code and thought processes. In our component equal to your GameBoard, we made "winner" a sole component and called it EndGame.